src/library_wasmfs_opfs.js

```diff
@@ -425,11 +425,11 @@ mergeInto(LibraryManager.library, {
     wasmfsOPFSProxyFinish(ctx);
   },
 
-  _wasmfs_opfs_set_size_file__deps: ['$wasmfsOPFSFileHandles', '$wasmfsOPFSProxyFinish'],
+  _wasmfs_opfs_set_size_file__deps: ['$wasmfsOPFSFileHandles', '$wasmfsOPFSProxyFinish'].concat(i53ConversionDeps),
```

Comment:
Are there any other uses of `concat` to create deps? If it wouldn't be too complicated, it seems like it might be simpler to explicitly list them out.

Reply:
They might change over time, though? Unless we can find a way to avoid hard-coding the deps, I think I like @sbc100 's `concat` approach.

Reply:
The reason is mostly that they change depending on the configuration (i.e. `WASM_BIGINT` or not).  They entire purpose of `i53ConversionDeps` is to be `.concat`d like this.    We do it elsewhere already.. this one was just overlooked